src/app/modules/group/pages/current-user/current-user.component.ts

```diff
@@ -36,4 +37,19 @@ export class CurrentUserComponent {
     });
   }
 
+  onModify(userId: string): void {
+    window.open(
+      `${ environment.oauthServerUrl }?all=1&client_id=${ userId }&redirect_uri=${encodeURI(`${ location.origin }/update-profile.html`)}`,
```

Comment:
I think this works in localhost but won't work in production. Encoded redirect uri will be:
- on localhost: `'http://localhost:4200/update-profile.html'`
- on production algorea: `'https://dev.algorea.org/update-profile.html'` instead of `'https://dev.algorea.org/{en | fr}/update-profile.html'`

I tested, on your branch the html file is located at
`'https://dev.algorea.org/branch/feature/let-user-modify-his-profile/{en | fr}/update-profile.html'`
so in production the file will be located at
`https://dev.algorea.org/{en | fr}/update-profile.html`

If we want to make it context agnostic, we can use a relative path:
```js
const url = new URL(
  './update-profile.html' /* relative url */,
  location.href /* base of relative url */
).href;
```
That way it will work for branch build, production builds, localhost and fr/en builds.

Reply:
Fixed